worldedit-core/src/main/java/com/sk89q/worldedit/command/SelectionCommands.java

```diff
@@ -105,6 +105,55 @@ public SelectionCommands(WorldEdit we) {
         this.we = we;
     }
 
+    @Command(
+        name = "/pos",
+        desc = "Set positions"
+    )
+    @Logging(POSITION)
+    @CommandPermissions("worldedit.selection.pos")
+    public void pos(Actor actor, World world, LocalSession session,
+                    @Arg(desc = "Coordinates to set position 1 to", def = "")
+                        BlockVector3 pos1,
+                    @Arg(desc = "Coordinates to set position 2 to", def = "")
+                        BlockVector3 pos2,
+                    @Arg(desc = "Coordinates to add as extras if selection type supports", def = "", variable = true, name = "pos2")
+                        List<BlockVector3> pos2Extra) throws WorldEditException {
+        if (pos1 == null) {
+            if (actor instanceof Locatable) {
+                pos1 = ((Locatable) actor).getBlockLocation().toVector().toBlockPoint();
+            } else {
+                actor.printError(TranslatableComponent.of("worldedit.pos.console-require-coords"));
+                return;
+            }
+        }
+
+        if (pos2 == null) {
+            if (actor instanceof Locatable) {
+                pos2 = ((Locatable) actor).getBlockLocation().toVector().toBlockPoint();
+            } else {
+                actor.printError(TranslatableComponent.of("worldedit.pos.console-require-coords"));
+                return;
+            }
+        }
+
+        session.getRegionSelector(world).selectPrimary(pos1, ActorSelectorLimits.forActor(actor));
+        session.getRegionSelector(world).selectSecondary(pos2, ActorSelectorLimits.forActor(actor));
+
+        if (pos2Extra != null && !pos2Extra.isEmpty()) {
+            for (BlockVector3 vector : pos2Extra) {
+                session.getRegionSelector(world).selectSecondary(vector, ActorSelectorLimits.forActor(actor));
+            }
+        }
+
+        session.getRegionSelector(world)
+            .explainRegionAdjust(actor, session);
```

Comment:
```suggestion
        session.getRegionSelector(world).explainRegionAdjust(actor, session);
```
Not sure why you have a newline here.